crates/net/eth-wire/src/handshake.rs

```diff
@@ -110,7 +111,7 @@ where
             }
         };
 
-        if their_msg.len() > MAX_STATUS_SIZE {
+        if their_msg.len() > MAX_MESSAGE_SIZE {
```

Comment:
yeah I guess it doesnt really matter what value we check here because at this point we've already read the entire thing, could also just check for something smaller because this should just contain the status, but doesnt really matter, malformed status would just fail in the decoding step